pkg/credentials/matcher/matcher.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2024 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package matcher
+
+import (
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+// VolumePath is the path where build secrets are written.
+// It is mutable and exported for testing.
+var VolumePath = "/tekton/creds-secrets"
+
+// Builder is the interface for a credential initializer of any type.
+type Builder interface {
```

Comment:
I wonder if `Builder` is the correct name now 😛 Could be `Matcher` or something ? (I am bad at naming stuff)

Reply:
sending the updates noww